numpy/distutils/system_info.py

```diff
@@ -489,8 +495,9 @@ def parse_config_files(self):
                 self.cp.add_section(self.section)
 
     def calc_libraries_info(self):
-        libs = self.get_libraries()
-        dirs = self.get_lib_dirs()
+        libs   = self.get_libraries()
+        dirs   = self.get_lib_dirs()
```

Comment:
can you undo the added spaces here? PEP8 frowns upon such ``=`` alignment between lines